src/main/java/com/mackervoy/calum/mud/DatasetCollection.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * 
+ */
+package com.mackervoy.calum.mud;
+
+import java.io.File;
+
+/**
+ * @author Calum Mackervoy
+ * A collection (directory) of datasets. Manages the URI and file-store bindings to each particular
+ */
+public class DatasetCollection extends AbstractFileStorageWrapper {
+	
+	/**
+	 * @param subPath: the directory path of the collection *relative* the the root directory, starting and ending with a "/"
+	 * e.g. /world/mydatacollection/
+	 */
+	public DatasetCollection(String subPath) {
+		this.path = subPath;
```

Comment:
It looks like the functionality of `AbstractFileStorageWrapper` relies on this.path being defined so perhaps one to lift up to its constructor? As we do the same thing in `DatasetItem` as well